app/helpers/report_helper.rb

```diff
@@ -137,13 +137,15 @@ def populate_reports_menu(hide_custom = false)
   end
 
   def reports_group_title
-    tenant_name = User.current_tenant.name
-    if User.current_user.report_admin_user?
+    tenant = User.current_tenant
+    user = User.current_user
+    tenant_name = tenant ? tenant.name : ''
+    if user && user.report_admin_user?
```

Comment:
I don't really undersatnd the need for these changes.

Reply:
Hi,
While converting a rails table from the file `_report_list.html.haml` a new component was created named '`ReportList`'.

After pushing the changes, a travis check failed as shown in the below screenshot.
Command : `bundle exec rspec spec/views/report/_report_list.html.haml_spec.rb`
![image](https://user-images.githubusercontent.com/87487049/139686858-cfd70199-41d9-4631-8e8f-a6e8f20d158a.png)

Reply:
I'm not sure this fix is proper then, because all it does is bypass a missing user/tenant, but there shouldn't be a missing user/tenant.  This is important for RBAC reasons, so we shouldn't patch around it unless we understand why the user/tenant is suddenly gone.

Reply:
@jeffibm If i declare dummy variable `user` in _report_list.html.haml_spec.rb, testcase is working for me. You need not change anything in helper file.

```
  context 'check links in the list view' do
    let(:user) { FactoryBot.create(:user_with_group) }

    before do
      login_as user
    end

    it 'check links in the list view' do
        render
        expect(response).to have_selector("//div[@id=\"report_list_div\"]")
    end
  end